examples/program/WORKSPACE

```diff
@@ -0,0 +1,21 @@
+workspace(name = "program_example")
```

Comment:
Better name than program?  testable?

Reply:
I'm planning to extend this example to cover a couple other missing things related to running code that you author under nodejs_binary (to reproduce problems currently seen only in repos that consume this) so I think this name will be okay.